components/Menu.vue

```diff
@@ -1,313 +1,21 @@
 <template>
-  <div>
-    <nav flex space-between align-center>
-      <div flex row space-between align-center>
-        <div>
-          <nuxt-link :to="links.home">
-            <img src="~/assets/MESG-logo-horizontal-purple.svg" alt="MESG" />
-          </nuxt-link>
-        </div>
-        <a @click="open = !open" class="burger" hidden-desktop>
-          <span v-if="open">&times;</span>
-          <span v-else>☰</span>
-        </a>
-      </div>
-      <div flex row space-between class="actions" :class="{open}">
-        <ul flex row mobile-column>
-          <li class="drop-down" flex column align-center>
-            <a href="#" class="top-menu">
-              Technology
-              <i class="far fa-angle-down"></i>
-            </a>
-            <div flex column class="sub-menu" p1 mt1>
-              <nuxt-link :to="links.orchestrator">Orchestrator</nuxt-link>
-              <nuxt-link :to="links.engine">SDK</nuxt-link>
-              <nuxt-link :to="links.marketplace">Marketplace</nuxt-link>
-              <nuxt-link :to="links.enterprise">Enterprise</nuxt-link>
-            </div>
-          </li>
-          <li class="drop-down" flex column align-center>
-            <a href="#" class="top-menu">
-              Developers
-              <i class="far fa-angle-down"></i>
-            </a>
-            <div flex column class="sub-menu" p1 mt1>
-              <a :href="externalLinks.getStarted" target="_blank">Get started</a>
-              <a :href="externalLinks.marketplace" target="_blank">Marketplace</a>
-              <a :href="externalLinks.documentation" target="_blank">Documentation</a>
-              <a :href="externalLinks.tutorials" target="_blank">Tutorials</a>
-              <a :href="externalLinks.github" target="_blank">Github</a>
-            </div>
-          </li>
-          <li flex align-center>
-            <nuxt-link :to="links.showcase" class="top-menu">Showcase</nuxt-link>
-          </li>
-          <li flex align-center>
-            <nuxt-link :to="links.contributions" class="top-menu">Contributions</nuxt-link>
-          </li>
-          <li flex align-center>
-            <nuxt-link :to="links.token" class="top-menu">Token</nuxt-link>
-          </li>
-          <li flex align-center>
-            <Button primary small :to="links.getstarted">Get started</Button>
-          </li>
-        </ul>
-      </div>
-    </nav>
-  </div>
+  <Menu :logo="require('~/assets/MESG-logo-horizontal-purple.svg')" :items="menu" is-nuxt />
```

Comment:
Can you just replace the menu where it's used (the layout) like that you can delete this component? It's weird to have a component menu that includes a menu.

Reply:
Done